backends/advanced/src/advanced_omi_backend/services/transcription/__init__.py

```
@@ -1,120 +1,269 @@
+
+        final = None
+        try:
+            # Drain until final or close
+            for _ in range(500):  # hard cap
+                msg = await asyncio.wait_for(ws.recv(), timeout=1.5)
+                data = json.loads(msg)
+                if not final_type or data.get("type") == final_type:
+                    final = data
+                    break
+        except Exception:
+            pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)